lib/burials/monitor.rb

```diff
@@ -39,8 +44,13 @@ def track_show404(confirmation_number, current_user, e)
     # @param e [Error]
     #
     def track_show_error(confirmation_number, current_user, e)
-      Rails.logger.error('21P-530EZ fetching submission failed',
-                         { confirmation_number:, user_uuid: current_user&.uuid, message: e&.message })
+      additional_context = {
+        confirmation_number:,
+        user_uuid: current_user&.uuid,
+        message: e&.message
+      }
+      track_request('error', '21P-530EZ fetching submission failed', CLAIM_STATS_KEY, additional_context,
```

Comment:
@wayne-weibel Wouldn't we rather it just be `nil` over some arbitrary String?

Reply:
Sorry for the ambiguity - MESSAGE == the current log message.  I also think the message could be generalized to use a variable instead of the hard coded form type, ie 21P-530EZ, 21P-527EZ, etc

Reply:
@wayne-weibel We don't get the form_type in the arguments to use it as a variable. I feel like `message` in the payload should just be the actual error message and then we send our hard-coded message in as well instead of them being duplicated.

Reply:
the form_type could be part of the initialize method, and then possibly abstract the functions further, but we can table that for later.  not needed for this ticket now